javascript/stationBoard-view.js

```diff
@@ -88,8 +88,25 @@ let stationBoardView = {
 		let type = document.createElement("td");
 		type.classList.add("type");
 
-		type.appendChild(document.createTextNode(train.lineName));
-
+        let journey;
```

Comment:
please put new functionality into separte method.

Reply:
Added the method, c3808ab.